# Uighur translation for apport
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-09-10 11:16+0200\n"
"PO-Revision-Date: 2015-04-27 21:46+0000\n"
"Last-Translator: bilgeturghun <bilgeks@gmail.com>\n"
"Language-Team: Uighur <ug@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2015-09-29 05:10+0000\n"
"X-Generator: Launchpad (build 17768)\n"
"Language: ug\n"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:430 ../kde/apport-kde.py:466
msgid "Apport"
msgstr "Apport"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:2 ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "ئەمەلدىن قالدۇر"

#: ../debian/tmp/usr/share/apport/apport-gtk.ui.h:3 ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "تامام"

#: ../gtk/apport-gtk.ui.h:2
msgid "Crash report"
msgstr "يىمىرىلىش دوكلاتى"

#: ../gtk/apport-gtk.ui.h:3
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>كەچۈرۈڭ، ئىچكى خاتالىق يۈز بەردى.</b></big>"

#: ../gtk/apport-gtk.ui.h:4 ../gtk/apport-gtk.py:233 ../kde/apport-kde.py:213
msgid "If you notice further problems, try restarting the computer."
msgstr ""
"ئەگەر تېخىمۇ كۆپ مەسىلىنى بايقىسىڭىز، كومپيۇتېرنى قايتا قوزغىتىشنى سىناڭ."

#: ../gtk/apport-gtk.ui.h:5
msgid "Send an error report to help fix this problem"
msgstr ""
"بۇ مەسىلىنى ئوڭلاشقا ياردەم قىلىش ئۈچۈن خاتالىق دوكلاتىدىن بىرنى يوللاڭ"

#: ../gtk/apport-gtk.ui.h:6
msgid "Ignore future problems of this program version"
msgstr "بۇ پروگرامما نەشرىنىڭ كەلگۈسىدىكى مەسىلىلىرىگە پەرۋا قىلما"

#: ../gtk/apport-gtk.ui.h:7 ../gtk/apport-gtk.py:172 ../gtk/apport-gtk.py:497
#: ../kde/apport-kde.py:255
msgid "Show Details"
msgstr "تەپسىلاتلارنى كۆرسەت"

#: ../gtk/apport-gtk.ui.h:8
msgid "_Examine locally"
msgstr "يەرلىكتە تەكشۈر(_E)"

#: ../gtk/apport-gtk.ui.h:9 ../gtk/apport-gtk.py:221 ../kde/apport-kde.py:203
msgid "Leave Closed"
msgstr "تاقاق پېتى قالدۇر"

#: ../gtk/apport-gtk.ui.h:10 ../gtk/apport-gtk.py:184 ../gtk/apport-gtk.py:225
#: ../gtk/apport-gtk.py:236 ../kde/apport-kde.py:207 ../kde/apport-kde.py:216
msgid "Continue"
msgstr "داۋاملاشتۇر"

#: ../gtk/apport-gtk.ui.h:11
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>مەسىلە ھەققىدە ئۇچۇرلارنى توپلاۋاتىدۇ</b></big>"

#: ../gtk/apport-gtk.ui.h:12
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"سىز مەلۇم قىلغان مەسىلىنى ئىجادىيەتچىلەرنىڭ تۈزىتىشىگە ياردىمى بولسۇن دەپ "
"ئۇچۇرلار توپلاندى."

#: ../bin/apport-cli.py:246 ../gtk/apport-gtk.ui.h:13 ../kde/apport-kde.py:395
msgid "Uploading problem information"
msgstr "مەسىلىلەر ھەققىدىكى ئۇچۇرلارنى يېڭىلاۋاتىدۇ"

#: ../gtk/apport-gtk.ui.h:14
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>مەسىلە ھەققىدىكى ئۇچۇرلارنى يۈكلەۋاتىدۇ</b></big>"

#: ../gtk/apport-gtk.ui.h:15 ../kde/apport-kde.py:396
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"توپلانغان ئۇچۇرلارنى كەمتۈك باشقۇرۇش سىستېمىسىغا ئەۋەتمەكچى. بۇنىڭغا ئازراق "
"ۋاقىت كېتىدۇ."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Apport چاتاق ھۆججىتى"

#: ../bin/apport-cli.py:143 ../gtk/apport-gtk.py:118 ../kde/apport-kde.py:322
msgid "(binary data)"
msgstr "(ئىككىلىك سانلىق-مەلۇمات)"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:157 ../gtk/apport-gtk.py:157
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "كەچۈرۈڭ، پروگرامما %s  تاسادىپىي توختاپ قالدى."

#: ../gtk/apport-gtk.py:136
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "كەچۈرۈڭ، %s تاسادىپىي تاقالدى."

#: ../gtk/apport-gtk.py:142 ../kde/apport-kde.py:177 ../kde/apport-kde.py:210
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "كەچۈرۈڭ، %s ئىچكى خاتالىققا يولۇقتى."

#: ../bin/apport-cli.py:173 ../gtk/apport-gtk.py:154 ../kde/apport-kde.py:163
msgid "Send problem report to the developers?"
msgstr "بۇ مەلۇماتنى ئىجادىيەتچىگە ئەۋەتسۇنمۇ؟"

#: ../gtk/apport-gtk.py:163 ../kde/apport-kde.py:171
msgid "Send"
msgstr "يوللا"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:228 ../gtk/apport-gtk.py:228
msgid "Force Closed"
msgstr "مەجبۇرىي يېپىلغان"

#: ../gtk/apport-gtk.py:222 ../kde/apport-kde.py:204 ../kde/apport-kde.py:341
msgid "Relaunch"
msgstr "قايتا ئىجرا قىل"

#: ../debian/tmp/usr/share/apport/apport-gtk.py:238 ../gtk/apport-gtk.py:238
#, python-format
msgid "The application %s has stopped responding."
msgstr "پروگرامما %s جاۋاب قايتۇرۇشتىن توختىدى."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:242 ../gtk/apport-gtk.py:242
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "پروگرامما «%s» جاۋاب قايتۇرۇشتىن توختىدى."

#: ../gtk/apport-gtk.py:200 ../kde/apport-kde.py:185
#, python-format
msgid "Package: %s"
msgstr "بوغچا: %s"

#: ../gtk/apport-gtk.py:207 ../kde/apport-kde.py:191
msgid "Sorry, a problem occurred while installing software."
msgstr "كەچۈرۈڭ، يۇمشاق دېتال ئورناتقاندا بىر خاتالىق كۆرۈلدى."

#: ../debian/tmp/usr/share/apport/apport-gtk.py:273
#: ../debian/tmp/usr/share/apport/apport-gtk.py:292 ../gtk/apport-gtk.py:273
#: ../gtk/apport-gtk.py:292 ../debian/tmp/usr/share/apport/apport-kde.py:219
#: ../kde/apport-kde.py:219
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "پروگرامما %s نىڭدا بىر ئىچكى خاتالىق يۈز بەردى."

#: ../gtk/apport-gtk.py:216 ../kde/apport-kde.py:197
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "بۇ %s ئەپ تاسادىپىي تاقالدى."

#: ../gtk/apport-gtk.py:237 ../kde/apport-kde.py:217
msgid "Ignore future problems of this type"
msgstr "بۇ تۈردىكى كەلگۈسىدىكى مەسىلىگە سەل قاراش"

#: ../gtk/apport-gtk.py:501 ../kde/apport-kde.py:252
msgid "Hide Details"
msgstr "تەپسىلاتلارنى يوشۇر"

#: ../bin/apport-unpack.py:23
#, python-format
msgid "Usage: %s <report> <target directory>"
msgstr "ئىشلىتىش ئۇسۇلى: %s <مەلۇمات> <نىشان مۇندەرىجە>"

#: ../bin/apport-unpack.py:33
msgid "Destination directory exists and is not empty."
msgstr "نىشان مۇندەرىجە بار ھەم بوش ئەمەس."

#: ../kde/apport-kde.py:278
msgid "Username:"
msgstr "ئىشلەتكۈچى ئاتى:"

#: ../kde/apport-kde.py:279
msgid "Password:"
msgstr "ئىم:"

#: ../kde/apport-kde.py:366
msgid "Collecting Problem Information"
msgstr "چاتاق ھەققىدىكى ئۇچۇرلارنى توپلاۋاتىدۇ"

#: ../bin/apport-cli.py:233 ../kde/apport-kde.py:367
msgid "Collecting problem information"
msgstr "مەسىلىلەرگە ئائىت ئۇچۇرلارنى توپلاۋاتىدۇ"

#: ../kde/apport-kde.py:368
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"ئىجادىيەتچىنىڭ پروگرامما ئىقتىدارىنى ياخشىلىشىغا قۇلايلىق يارىتىش ئۈچۈن، "
"توپلانغان ئۇچۇرلار ئىجادىيەتچىگە يوللىنىدۇ. بۇنىڭغا بىرقانچە مىنۇت كېتىدۇ."

#: ../kde/apport-kde.py:394
msgid "Uploading Problem Information"
msgstr "چاتاق ھەققىدىكى ئۇچۇرلارنى يۈكلەۋاتىدۇ"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "سىستېما ئۇچۇرلىرىنى توپلا"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"بۇ مەسىلىنى مەلۇم قىلىشتا سىستېما ئۇچۇرلىرىنى توپلاش كېرەك، بىراق بۇنىڭ "
"ئۈچۈن كىملىك دەلىللەش زۆرۈردۇر."

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "سىستېما مەسىلە دوكلاتى"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid ""
"Please enter your password to access problem reports of system programs"
msgstr "ئىمنى كىرگۈزۈپ سىستېما مەسىلە دوكلاتىنى كۆرۈڭ."

#: ../gtk/apport-gtk-mime.desktop.in.h:1 ../kde/apport-kde.desktop.in.h:1
#: ../kde/apport-kde-mime.desktop.in.h:1 ../gtk/apport-gtk.desktop.in.h:1
msgid "Report a problem..."
msgstr "مەسىلىنى مەلۇم قىلىش..."

#: ../gtk/apport-gtk-mime.desktop.in.h:2 ../kde/apport-kde.desktop.in.h:2
#: ../kde/apport-kde-mime.desktop.in.h:2 ../gtk/apport-gtk.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "خاتالىقنى ئىجادىيەتچىگە مەلۇم قىلىدۇ"

#: ../debian/tmp/usr/bin/apport-valgrind.py:37 ../bin/apport-valgrind.py:37
msgid "See man page for details."
msgstr "تېخىمۇ كۆپ تەپسىلاتنى man قوللانمىسىدىن كۆرۈڭ."

#: ../debian/tmp/usr/bin/apport-valgrind.py:43 ../bin/apport-valgrind.py:43
msgid "specify the log file name produced by valgrind"
msgstr "بۇ valgrind ھاسىل قىلىدىغان خاتىرە ھۆججىتىنىڭ ئىسمىنى بەلگىلەڭ."

#: ../debian/tmp/usr/bin/apport-valgrind.py:46 ../bin/apport-valgrind.py:46
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"ئىلگىرى قۇرۇلغان sandbox مۇندەرىجىسى (SDIR) نى قايتا ئىشلىتىدۇ ياكى ئەگەر ئۇ "
"مەۋجۇت بولمىسا، ئۇنى قۇرىدۇ"

#: ../debian/tmp/usr/bin/apport-valgrind.py:50 ../bin/apport-valgrind.py:50
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"قوشۇمچە سازلاش بەلگىسى ئۈچۈن sandbox مۇندەرىجىسىنى قۇرمايدۇ ياكى قايتا "
"ئىشلەتمەيدۇ ئەمما پەقەت ئورنىتىلغان سازلاش بەلگىلىرىگە تايىنىدۇ."

#: ../debian/tmp/usr/bin/apport-valgrind.py:54 ../bin/apport-valgrind.py:54
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"ئىلگىرى قۇرۇلغان غەملەك مۇندەرىجە (CDIR) نى قايتا ئىشلىتىدۇ ياكى ئەگەر "
"مەۋجۇت بولمىسا ئۇنى قۇرىدۇ"

#: ../debian/tmp/usr/bin/apport-valgrind.py:58 ../bin/apport-valgrind.py:58
msgid ""
"report download/install progress when installing packages into sandbox"
msgstr ""
"ئورنىتىش بوغچىسى sandbox قا كىرسە چۈشۈرۈش/ئورنىتىش جەريانىنى دوكلات قىلىدۇ"

#: ../debian/tmp/usr/bin/apport-valgrind.py:62 ../bin/apport-valgrind.py:62
msgid ""
"the executable that is run under valgrind's memcheck tool  for memory leak "
"detection"
msgstr ""
"ئەسلەك ھالقىشنى بايقاش ئۈچۈن ئىجراچان پىروگراممىلار valgrind ئەسلەك تەكشۈرۈش "
"قورالىنىڭ ئاستىدا ئىجرا قىلىنىدۇ"

#: ../bin/apport-retrace.py:63
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""
"قۇم خالتىسى ئىچىگە قوشۇمچە بوغچىنى ئورنىتىش(بىر قانچە قېتىم كۆرسىتىشكە "
"بولىدۇ)"

#: ../debian/tmp/usr/bin/apport-valgrind.py:97 ../bin/apport-valgrind.py:97
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "خاتالىق: %s نى ئىجرا قىلغىلى بولمايدۇ. توختىتىۋاتىدۇ."

#: ../data/kernel_oops.py:28
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"سىستېمىڭىزنىڭ تۇراقسىز بولۇپ قېلىش ئېھتىماللىقى بار، قايتا قوزغىتىش "
"زۆرۈرىيىتى باردەك قىلىدۇ."

#: ../bin/apport-retrace.py:39
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "بۇ يېڭى چىققانلىرىنى مەلۇماتقا يازماي stdout قا يازسۇن."

#: ../bin/apport-retrace.py:41
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"مەلۇماتنىڭ غول تۆكمىسىگە نىسبەتەن تەسىرلىشىشچان gdb ئەڭگىمەسىنى باشلايدۇ(-o "
"غا پەرۋا قىلمايدۇ؛ مەلۇماتنى قاپلىمايدۇ)"

#: ../bin/apport-retrace.py:43
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr "ئەسلىدىكى مەلۇماتقا يازماي كۆرسىتىلگەن ھۆججەتكە يازسۇن."

#: ../bin/apport-retrace.py:45
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"stack ئىزلىشى قايتا ھاسىللانغاندىن كېيىن، غول تۆكمىنى(core dump) مەلۇماتتىن "
"چىقىرىۋېتىش"

#: ../bin/apport-retrace.py:47
msgid "Override report's CoreFile"
msgstr "دوكلاتنىڭ CoreFile نى قاپلاش"

#: ../bin/apport-retrace.py:49
msgid "Override report's ExecutablePath"
msgstr "دوكلاتنىڭ ExecutablePath نى قاپلاش"

#: ../bin/apport-retrace.py:51
msgid "Override report's ProcMaps"
msgstr "دوكلاتنىڭ ProcMaps نى قاپلاش"

#: ../bin/apport-retrace.py:53
msgid "Rebuild report's Package information"
msgstr "دوكلاتنىڭ بوغچا ئۇچۇرىنى قايتا قۇرۇش"

#: ../bin/apport-retrace.py:55
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"ۋاقتىنچە قۇم ساندۇقى قۇرىدۇ ھەم قاچىلاش بوغچىسىنى يۈكلەيدۇ/قاچىلايدۇ.ئەگەر "
"بۇ تاللانما ئىشلىتىلمىسە، سىستېمىغا لازىملىق دېتالنى قاچىلايدۇ. ئەگەر سىز "
"سىستېما بەلگىلىمىسىڭىز، سىستېما سەپلىمىسىنى ئىشلىتىدۇ، شۇنىڭ بىلەن ئىجرا "
"قىلىنىۋاتقان سىستېمىنى ھالاك قىلىدۇ."

#: ../bin/apport-retrace.py:57
msgid ""
"Report download/install progress when installing packages into sandbox"
msgstr ""
"بوغچىلارنى قۇم خالتىسى ئىچىگە ئورناتقاندىكى چۈشۈرۈش/ئورنىتىش جەريانىنى مەلۇم "
"قىلىش"

#: ../bin/apport-retrace.py:59
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""
"ئۇچۇر خاتىرىلەشتىن بۇرۇن ۋاقىتنى خاتىرىلەڭ، بۇ قېتىملىق مەشغۇلاتنى "
"تەمىنلەيدۇ."

#: ../debian/tmp/usr/bin/apport-retrace.py:57 ../bin/apport-retrace.py:57
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""

#: ../bin/apport-retrace.py:61
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "قۇم خالتىسى ئىچىگە چۈشۈرۈلگەن بوغچىلارنىڭ غەملەك مۇندەرىجىسى"

#: ../debian/tmp/usr/bin/apport-retrace.py:61 ../bin/apport-retrace.py:61
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"يېيىلمىغان بوغچىلارنىڭ مۇندەرىجىسى. كېيىن ئىجرا قىلىدىغان چاغدا، چۈشۈرۈلگەن "
"بوغچىلار مۇشۇ يەرگە يېيىلىدۇ."

#: ../bin/apport-retrace.py:65
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"يىمىرىلىش ساندانىنى زىيارەت قىلىش ئۇچۇرى ھۆججىتىنىڭ يولىنى ئۆز ئىچىگە "
"ئالىدۇ. بۇ تاللانما مۇئەييەن يىمىرىلىش كودىدا  ئىزلاشنىڭ قايتۇرما سازلاش "
"ئۇچۇرى ئۈچۈن ئىشلىتىلىدۇ. (ئىشلەتكۈچى -g, -o, ياكى -s نى بەلگىلىگەن "
"بولمىسىلا)"

#: ../bin/apport-retrace.py:67
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"قايتۇرغان ئىزلاش ئۇچۇرىنى كۆرسىتىدۇ ھەمدە يىمىرىلىش ساندىنىغا يوللاش "
"يوللىماسلىقنى جەزملەشنى سورايدۇ."

#: ../bin/apport-retrace.py:69
msgid ""
"Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"تەكرارلانغان sqlite ساندىنىنىڭ يولى.(كۆڭۈلدە تەكرارلىنىشنى تەكشۈرمەيدۇ)"

#: ../debian/tmp/usr/bin/apport-retrace.py:78 ../bin/apport-retrace.py:78
msgid "You cannot use -C without -S. Stopping."
msgstr ""
"سىز -S نى بەلگىلىمەي تۇرۇپ -C تاللانمىنى ئىشلىتەلمەيسىز، توختىتىۋاتىدۇ."

#. translators: don't translate y/n, apport currently only checks for "y"
#: ../bin/apport-retrace.py:110
msgid "OK to send these as attachments? [y/n]"
msgstr "بۇ قوشۇلما ھۆججەتلەرنى ئەۋەتسە بولامدۇ؟[y/n]"

#: ../apport/ui.py:92
msgid "This package does not seem to be installed correctly"
msgstr "مەزكۇر بوغچا توغرا ئورنىتىلمىغاندەك قىلىدۇ"

#. TRANS: %s is the name of the operating system
#: ../apport/ui.py:95
#, python-format
msgid ""
"This is not an official %s package. Please remove any third party package "
"and try again."
msgstr ""
"بۇ رەسمىي %s بوغچىسى ئەمەس. ئۈچىنچى تەرەپ بوغچىسىنى چىقىرىۋېتىپ، قايتا سىناڭ."

#: ../apport/ui.py:115
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"بەزىبىر كونا بوغچىلار بار ئىكەن. تۆۋەندىكى بوغچىلارنى يۈكسەلدۈرگەندىن كېيىن "
"يەنە تەكشۈرۈپ كۆرۈڭ:\n"
"\n"
"%s"

#: ../apport/ui.py:211
msgid "unknown program"
msgstr "نامەلۇم پروگرامما"

#: ../apport/ui.py:212
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "كەچۈرۈڭ، پروگرامما  «%s» ئويلىمىغان يەردىن تاقىلىپ قالدى"

#: ../apport/ui.py:213 ../apport/ui.py:1118
#, python-format
msgid "Problem in %s"
msgstr "مەسىلە %s نىڭدا"

#: ../apport/ui.py:214
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"مەسىلىنى ئاپتوماتىك ئانالىز قىلىش ۋە ئىجادىيەتچىگە مەلۇمات يوللاشقا كېرەكلىك "
"ئەسلەك يېتەرلىك ئەمەس ئىكەن."

#. package does not exist
#: ../apport/ui.py:228 ../apport/ui.py:235 ../apport/ui.py:241
#: ../apport/ui.py:284 ../apport/ui.py:292 ../apport/ui.py:363
#: ../apport/ui.py:366 ../apport/ui.py:564 ../apport/ui.py:933
#: ../apport/ui.py:1072 ../apport/ui.py:1076 ../apport/ui.py:1095
#: ../apport/ui.py:1101
msgid "Invalid problem report"
msgstr "مەلۇمات ئىناۋەتسىز"

#: ../apport/ui.py:285
msgid "You are not allowed to access this problem report."
msgstr "بۇ مەسىلىنى مەلۇم قىلىش ھوقۇقىڭىز يوق."

#: ../apport/ui.py:288
msgid "Error"
msgstr "خاتالىق"

#: ../apport/ui.py:289
msgid "There is not enough disk space available to process this report."
msgstr "مەزكۇر مەلۇماتنى بىر تەرەپ قىلىشقا كېرەكلىك دىسكا بوشلۇقى يوق."

#: ../apport/ui.py:316
msgid "No package specified"
msgstr "بوغچا كۆرسىتىلمىگەن."

#: ../apport/ui.py:317
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"بوغچا ياكى بىر PID نى كۆرسىتىڭ. تەپسىلاتلار ئۈچۈن --help نى ئىشلىتىڭ."

#: ../apport/ui.py:342
msgid "Permission denied"
msgstr "ھوقۇقىڭىز يەتمىدى"

#: ../apport/ui.py:343
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"بۇ سىزنىڭ پروگراممىڭىز ئەمەس. مەزكۇر پروگراممىنى ئۆزىڭىزنىڭ ياكى root نىڭ "
"ھوقۇقىدا ئىجرا قىلدۇرۇڭ."

#: ../apport/ui.py:333
msgid "Invalid PID"
msgstr "ئىناۋەتسىز PID"

#: ../apport/ui.py:334
msgid "The specified process ID does not belong to a program."
msgstr "كۆرسىتىلگەن PID مەزكۇر پروگراممىنىڭ PID ئەمەس."

#: ../apport/ui.py:364
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "ئەھۋال قوليازما %s تەسىر كۆرسىتىدىغان بوغچىنى بەلگىلىمىگەن"

#: ../apport/ui.py:367
#, python-format
msgid "Package %s does not exist"
msgstr "بوغچا %s مەۋجۇت ئەمەس"

#: ../apport/ui.py:392 ../apport/ui.py:576 ../apport/ui.py:581
msgid "Cannot create report"
msgstr "مەلۇمات قۇرالمىدى"

#: ../apport/ui.py:407 ../apport/ui.py:453 ../apport/ui.py:470
msgid "Updating problem report"
msgstr "مەلۇماتىنى يېڭىلاۋاتىدۇ"

#: ../apport/ui.py:408
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"سىز بۇ مەسىلىنى مەلۇم قىلغان كىشى ئەمەس، ياكى مەلۇمات تەكرارلانغان ياكى "
"ئاللىقاچان تاقالغان.\n"
"\n"
"«apport-bug» نى ئىشلىتىپ يېڭىدىن مەلۇمات تەييارلاڭ."

#: ../apport/ui.py:417
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"سىز بۇ مەسىلىنى مەلۇم قىلغۇچى ئەمەس. يېڭى كەمتۈك دەپ مەلۇم قىلغاندىن كۆرە، "
"باشقا كەمتۈك بىلەن ئوخشاش دەپ بەلگە قويۇش تېخىمۇ ئاسان.\n"
"\n"
"ئارقىدىن، «apport-bug» نى ئىشلىتىپ، كەمتۈك مەلۇم قىلىپ، مەلۇماتنىڭ ئىچىگە "
"كەمتۈك ھەققىدىكى ئىزاھاتنى يېزىشىڭىزنى تەۋسىيە قىلىمىز.\n"
"\n"
"راستلا داۋاملاشتۇرامسىز؟"

#: ../apport/ui.py:454 ../apport/ui.py:471
msgid "No additional information collected."
msgstr "قوشۇمچە ئۇچۇرلار توپلانمىدى."

#: ../apport/ui.py:521
msgid "What kind of problem do you want to report?"
msgstr "قانداق مەسىلىنى مەلۇم قىلماقچى؟"

#: ../apport/ui.py:538
msgid "Unknown symptom"
msgstr "نامەلۇم ھادىسە"

#: ../apport/ui.py:539
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "ھادىسە «%s» نامەلۇم."

#: ../apport/ui.py:567
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"ئۇچۇرنى ياپقاندىن كېيىن پروگرامما كۆزنىكىنى چېكىپ، بۇ مەسىلىنى مەلۇم قىلىڭ."

#: ../apport/ui.py:577 ../apport/ui.py:582
msgid "xprop failed to determine process ID of the window"
msgstr "xprop كۆزنەكنىڭ ئىجرا ID سىنى تېپىشتا مەغلۇپ بولدى"

#: ../apport/ui.py:596
msgid "%prog <report number>"
msgstr "%prog <مەلۇمات نومۇرى>"

#: ../apport/ui.py:598
msgid "Specify package name."
msgstr "بوغچا ئاتىنى كۆرسىتىپ بېرىڭ."

#: ../apport/ui.py:600 ../apport/ui.py:649
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "دوكلاتقا يېڭىدىن خەتكۈش قوشۇڭ، كۆپ قېتىم بەلگىلەشكە بولىدۇ."

#: ../apport/ui.py:630
msgid ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%prog [options] [symptom|pid|package|program path|.apport/.crash file]"

#: ../apport/ui.py:633
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"كەمتۈك يېزىش ھالىتىدە باشلايدۇ. ئىجرا قىلغاندا --package ۋە --pid ياكى "
"پەقەتلا --pid تاللانمىسىنى يازسىلا بولىدۇ. ئەگەر ھېچقايسى يېزىلمىسا، "
"كۆرۈلگەن ئالامەتلەرنىڭ تىزىمىنى كۆرسىتىدۇ. (پەقەت يالغۇز ئارگۇمېنت "
"بېرىلگەندىلا.)"

#: ../apport/ui.py:635
msgid "Click a window as a target for filing a problem report."
msgstr "خاتالىق دوكلاتىنى تولدۇرۇش ئۈچۈن كۆزنەكنى نىشان قىلىپ چېكىڭ"

#: ../apport/ui.py:637
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"كەمتۈك يېڭىلاش ھالىتىدە قوزغىتىش. --package دېگەن تاللانمىنى ئىشلىتىشكە "
"بولىدۇ."

#: ../apport/ui.py:639
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"ھادىسە ھەققىدە مەلۇمات تەييارلاڭ. (Implied if symptom name is given as only "
"argument)"

#: ../apport/ui.py:641
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"--file-bug ھالىتىدە بوغچا ئاتىنى كۆرسىتىڭ. ئەگەر --pid كۆرسىتىلگەن بولسا، "
"بۇنى كۆرسەتمىسىمۇ بولىدۇ.(Implied if package name is given as only argument.)"

#: ../apport/ui.py:643
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"--file-bug  ھالىتىدە ئىجرا بولۇۋاتقان پروگراممىنى كۆرسىتىڭ. ئەگەر بۇ "
"كۆرسىتىلسە، كەمتۈك مەلۇماتىغا تېخىمۇ كۆپ ئۇچۇرلار قوشۇلىدۇ. (Implied if pid "
"is given as only argument.)"

#: ../debian/tmp/usr/lib/python2.7/dist-packages/apport/ui.py:742
#: ../apport/ui.py:742
#: ../debian/tmp/usr/lib/python3/dist-packages/apport/ui.py:742
msgid "The provided pid is a hanging application."
msgstr ""
"تەمىنلەنگەن pid بولسا بىر ئېسىلىپ قالغان(توختاپ قالغان) پروگرامما ئىكەن."

#: ../apport/ui.py:645
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"چاتاق مەلۇماتىنى  %s نىڭدىكى كۈتۈپ تۇرغان ھۆججەتنى ئەمەس بەلكى بېرىلگەن  "
".apport ياكى .crash ھۆججىتىنى ئىشلىتىپ يوللاش(پەقەت ھۆججەت ئاتى ئارگۇمېنت "
"شەكىلدە بېرىلگەندىلا.)"

#: ../apport/ui.py:647
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"كەمتۈك يېزىش ھالىتىدە، توپلانغان ئۇچۇرلارنى مەلۇم قىلماي ھۆججەتكە ساقلاپ "
"قويۇشقا، بۇ ھۆججەتنى كېيىن باشقا كومپيۇتېر ئارقىلىق يوللاشقىمۇ بولىدۇ."

#: ../apport/ui.py:651
msgid "Print the Apport version number."
msgstr "Apport نىڭ نەشر نومۇرىنى بېسىپ چىقىرىدۇ."

#: ../apport/ui.py:783
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"crash نى تەكشۈرۈش ئۈچۈن apport-retrace تېرمىنال كۆزنىكىدە ئىجرا بولىدۇ."

#: ../apport/ui.py:784
msgid "Run gdb session"
msgstr "gdb ئەڭگىمەسىنى ئىجرا قىلىش"

#: ../apport/ui.py:785
msgid "Run gdb session without downloading debug symbols"
msgstr "سازلاش بەلگىلىرىنى چۈشۈرمەيلا gdb ئەڭگىمەسىنى ئىجرا قىلىش"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:787
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "%s نى تولۇق بەلگىلەر بىلەن يېڭىلاش"

#: ../apport/ui.py:1091 ../apport/ui.py:1100
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr "بۇ دوكلات زادىلا ئورنىتىلىپ باقمىغان پروگراممىغا ئىشلىتىلىدۇ."

#: ../apport/ui.py:830
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr "توقۇنۇشتا ئۆزگەرگەن پروگرامما %s تە خاتالىق كۆرۈلدى."

#: ../apport/ui.py:230 ../apport/ui.py:1078
msgid "This problem report is damaged and cannot be processed."
msgstr "بوغچا بۇزۇلغان بولغاچقا بىر تەرەپ قىلغىلى بولمىدى."

#: ../apport/ui.py:236
msgid "The report belongs to a package that is not installed."
msgstr "مەلۇمات ئورنىتىلمىغان بوغچىغا تەۋە ئىكەن."

#: ../apport/ui.py:242
msgid "An error occurred while attempting to process this problem report:"
msgstr ""
"بۇ مەسىلە ھەققىدىكى مەلۇماتنى بىر تەرەپ قىلاي دېگەندە  خاتالىق يۈز بەردى:"

#: ../apport/ui.py:934
msgid "Could not determine the package or source package name."
msgstr "بوغچا ياكى مەنبە كود بوغچىسىنىڭ نامىنى بېكىتكىلى بولمىدى."

#: ../apport/ui.py:952
msgid "Unable to start web browser"
msgstr "توركۆرگۈنى قوزغاتقىلى بولمىدى"

#: ../apport/ui.py:953
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "%s نى كۆرىدىغان توركۆرگۈنى قوزغاتقىلى بولمىدى."

#: ../apport/ui.py:1028
#, python-format
msgid "Please enter your account information for the %s bug tracking system"
msgstr "كەمتۈك باشقۇرۇش سىستېمىسى %s غا ھېسابات ئۇچۇرىڭىزنى كىرگۈزۈڭ."

#: ../apport/ui.py:1039
msgid "Network problem"
msgstr "توردىكى مەسىلە"

#: ../apport/ui.py:1041
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr "crash ساندىنىغا باغلىنالمىدى، ئىنتېرنېت باغلىنىشىنى تەكشۈرۈپ كۆرۈڭ."

#: ../apport/ui.py:1067
msgid "Memory exhaustion"
msgstr "ئەسلەك يېتىشمىدى"

#: ../apport/ui.py:1068
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"سىستېمىڭىزدا مەزكۇر crash مەلۇماتىنى بىر تەرەپ قىلىشقا يېتەرلىك ئەسلەك يوق."

#: ../apport/ui.py:1121
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"مەسىلىنى مەلۇم قىلغىلى بولمىدى:\n"
"\n"
"%s"

#: ../apport/ui.py:1160 ../apport/ui.py:1167
msgid "Problem already known"
msgstr "مەسىلە ئاللىقاچان ئايدىڭلاشقان"

#: ../apport/ui.py:1161
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"توركۆرگۈدىكى كەمتۈك دوكلاتىدا بۇ مەسىلە ئاللىقاچان مەلۇم قىلىنغان. تەكشۈرۈپ "
"كۆرۈڭ، ئەگەر سىزنىڭ مەلۇماتىڭىزدا تېخىمۇ كۆپرەك ئۇچۇرلار بار بولسا "
"ئىجادىيەتچىلەر ئۈچۈن تېخىمۇ پايدىلىق بولاتتى."

#: ../apport/ui.py:1168
msgid "This problem was already reported to developers. Thank you!"
msgstr "مەسىلە ئىجادىيەتچىلەرگە مەلۇم قىلىندى. سىزگە تەشەككۈر!"

#: ../data/apportcheckresume.py:67
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:67
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:69
#: ../debian/tmp/usr/share/apport/apportcheckresume.py:69
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""

#: ../data/apportcheckresume.py:75
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"ئەسلىگە قايتۇرۇش باسقۇچى ئاخىرلىشاي دەپ قالدى، بۇنداق بولغاندا نورمال "
"تاماملىنىدۇ."

#: ../bin/apport-cli.py:71
msgid "Press any key to continue..."
msgstr "خالىغان كۇنۇپكىنى باسسىڭىز داۋاملىشىدۇ..."

#: ../bin/apport-cli.py:78
msgid "What would you like to do? Your options are:"
msgstr "نېمە قىلغۇڭىز بار؟ سىز تۆۋەندىكىلەرنى قىلالايسىز:"

#: ../bin/apport-cli.py:82
#, python-format
msgid "Please choose (%s):"
msgstr "تاللاڭ(%s):"

#: ../bin/apport-cli.py:141
#, python-format
msgid "(%i bytes)"
msgstr "(%i بايت)"

#: ../bin/apport-cli.py:174
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"مەلۇماتنى ئەۋەتىپ بولغاندىن كېيىن \n"
"توركۆرگۈدە ئېچىلغان بەتتىكى جەدۋەلنى تولدۇرۇڭ."

#: ../bin/apport-cli.py:177
#, python-format
msgid "&Send report (%s)"
msgstr "مەلۇمات (%s) نى ئەۋەت(&S)"

#: ../bin/apport-cli.py:181
msgid "&Examine locally"
msgstr "يەرلىكتە تەكشۈر(&E)"

#: ../bin/apport-cli.py:185
msgid "&View report"
msgstr "مەلۇمات كۆرسەت(&V)"

#: ../bin/apport-cli.py:186
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"كېيىن يەنە باشقىلارغا ئەۋەتىش ياكى كۆچۈرۈپ بېرىش ئۈچۈن مەلۇمات ھۆججىتى "
"ساقلاپ قوي(&K)"

#: ../bin/apport-cli.py:187
msgid "Cancel and &ignore future crashes of this program version"
msgstr ""
"پروگراممىنىڭ مەزكۇر نەشرىدە بۇنىڭدىن كېلىپ چىققان crashes نى ئەمەلدىن قالدۇر "
"ياكى پىسەنت قىلما(&I)"

#: ../bin/apport-cli.py:189 ../bin/apport-cli.py:266 ../bin/apport-cli.py:298
#: ../bin/apport-cli.py:319
msgid "&Cancel"
msgstr "ئەمەلدىن قالدۇر(&C)"

#: ../bin/apport-cli.py:217
msgid "Problem report file:"
msgstr "مەلۇمات ھۆججىتى:"

#: ../bin/apport-cli.py:223 ../bin/apport-cli.py:228
msgid "&Confirm"
msgstr "جەزملە(&C)"

#: ../bin/apport-cli.py:227
#, python-format
msgid "Error: %s"
msgstr "خاتالىق: %s"

#: ../bin/apport-cli.py:234
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"ئىجادىيەتچىلەرنىڭ پروگراممىنى مۇكەممەللەشتۈرۈشى ئۈچۈن \n"
"توپلانغان ئۇچۇرلارنى ئەۋەتىدۇ. بۇنىڭغا ئازراق ۋاقىت كېتىدۇ."

#: ../bin/apport-cli.py:247
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"توپلانغان ئۇچۇرلارنى كەمتۈك باشقۇرۇش سىستېمىسىغا ئەۋەتىدۇ.\n"
"بۇنىڭغا ئازراق ۋاقىت كېتىدۇ."

#: ../bin/apport-cli.py:297
msgid "&Done"
msgstr "تامام(&D)"

#: ../bin/apport-cli.py:303
msgid "none"
msgstr "يوق"

#: ../bin/apport-cli.py:304
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "تاللانغىنى : %s. كوپنى تاللاش:"

#: ../bin/apport-cli.py:320
msgid "Choices:"
msgstr "تاللاشلار:"

#: ../bin/apport-cli.py:334
msgid "Path to file (Enter to cancel):"
msgstr "ھۆججەتنىڭ يولى(بولدى قىلىش ئۈچۈن قايتۇرۇش كۇنۇپكىسىنى بېسىڭ):"

#: ../bin/apport-cli.py:340
msgid "File does not exist."
msgstr "ھۆججەت مەۋجۇت ئەمەس."

#: ../bin/apport-cli.py:342
msgid "This is a directory."
msgstr "بۇ بىر مۇندەرىجە."

#: ../bin/apport-cli.py:348
msgid "To continue, you must visit the following URL:"
msgstr "داۋاملاشتۇرۇش ئۈچۈن تۆۋەندىكى URL نى كۆرۈشىڭىز كېرەك:"

#: ../bin/apport-cli.py:350
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"ئەمدى توركۆرگۈنى قوزغىتىڭ ياكى، URL نى باشقا كومپيۇتېردىكى توركۆرگۈگە "
"كۆچۈرۈڭ."

#: ../bin/apport-cli.py:352
msgid "Launch a browser now"
msgstr "ھازىرلا توركۆرگۈنى قوزغىتىش"

#: ../bin/apport-cli.py:366
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"تېخى بىر تەرەپ قىلىنمىغان crash ھەققىدىكى مەلۇمات يوق. --help نى ئىشلەتسىڭىز "
"تېخىمۇ كوپ ئۇچۇرغا ئېرىشەلەيسىز."

#~ msgid "%prog [options] <apport problem report | crash ID>"
#~ msgstr "%prog [options] <apport problem report | crash ID>"

#~ msgid "incorrect number of arguments; use --help for a short help"
#~ msgstr ""
#~ "argument لارنىڭ سانى توغرا ئەمەس. --help نى ئىشلىتىپ قىسقىچە چۈشەندۈرۈشىنى "
#~ "كۆرۈڭ."

#, python-format
#~ msgid "Sorry, the program \"%s\" closed unexpectedly."
#~ msgstr "كەچۈرۈڭ، پروگرامما «%s» ئويلىمىغان يەردىن تاقىلىپ قالدى."

#, python-format
#~ msgid "Sorry, the package \"%s\" failed to install or upgrade."
#~ msgstr "كەچۈرۈڭ، بوغچا «%s» نى ئورنىتىش ياكى يۈكسەلدۈرۈش مەغلۇپ بولدى."

#~ msgid "Your system encountered a serious kernel problem."
#~ msgstr "سىستېمىڭىزنىڭ يادروسىدا(kernel) ئېغىر مەسىلە يۈز بەردى."

#~ msgid "Kernel problem"
#~ msgstr "يادرو(kernel) دىكى مەسىلە"

#~ msgid "Send this data to the developers?"
#~ msgstr "بۇ سانلىق-مەلۇماتلارنى ئىجادىيەتچىگە ئەۋەتسۇنمۇ؟"

#~ msgid ""
#~ "This occured during a previous suspend and prevented it from resuming "
#~ "properly."
#~ msgstr ""
#~ "بۇ ئالدىنقى قېتىملىق توڭلىتىش جەريانىدا يۈز بەرگەن بولۇپ، قايتا باشلاشقا "
#~ "بولمىدى."

#~ msgid ""
#~ "This occured during a previous hibernate and prevented it from resuming "
#~ "properly."
#~ msgstr ""
#~ "بۇ ئالدىنقى قېتىملىق ئۈچەككە كىرگۈزۈش جەريانىدا يۈز بەرگەن بولۇپ، قايتا "
#~ "باشلاشقا بولمىدى."

#, no-c-format
#~ msgid ""
#~ "<span weight=\"bold\" size=\"larger\">Sorry, the package \"%s\" failed to "
#~ "install or upgrade.</span>"
#~ msgstr ""
#~ "<span weight=\"bold\" size=\"larger\">كەچۈرۈڭ، بوغچا «%s» نى ئورنىتىش ياكى "
#~ "يۈكسەلدۈرۈش مەغلۇپ بولدى.</span>"

#~ msgid "Package problem"
#~ msgstr "بوغچىدىكى مەسىلە"

#, python-format
#~ msgid "This is not a genuine %s package"
#~ msgstr "بۇ رەسمىي  %s بوغچىسى ئەمەس"

#, python-format
#~ msgid "The package \"%s\" failed to install or upgrade."
#~ msgstr "بوغچا «%s» نى ئورنىتىش ياكى يېڭىلاش مەغلۇپ بولدى."

#~ msgid ""
#~ "you either need to do a local operation (-s, -g, -o) or supply an "
#~ "authentication file (--auth); see --help for a short help"
#~ msgstr ""
#~ "يەرلىكتە مەشغۇلات قىلىشىڭىز (-s، -g، -o) ياكى سالاھىيەت دەلىللەش ھۆججىتىنى (-"
#~ "-auth) تەمىنلىشىڭىز كېرەك. ئاددىي ياردەم ئۇچۇرىغا ئېرىشىش ئۈچۈن --help نى "
#~ "ئىشلىتىڭ."

#, python-format
#~ msgid "Sorry, the application %s has closed unexpectedly."
#~ msgstr "كەچۈرۈڭ بۇ %s ئەپ تاسادىپىي تاقالدى."

#~ msgid "Your system might become unstablenow and might need to be restarted."
#~ msgstr ""
#~ "سىستېمىڭىزنىڭ بىنورمال بولۇش ئېھتىماللىقى بار، شۇڭا قايتا قوزغىتىش زۆرۈر."
